src/widgets/table/table.rs

```diff
@@ -539,6 +539,8 @@ impl<'a> Table<'a> {
         self
     }
 
+    /// NOTE: segment_size is deprecated. Use flex method instead.
+    ///
```

Comment:
Don't we already have deprecated notice on the type? I don't think having this as an additional comment is not very useful. I'm not sure we have done the same thing for the other deprecated types though.

Reply:
Removed those two lines 👍🏻